### DISCUSSION TITLE
祝贺中国的朋友们国庆快乐! 🎉🎉🎉

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#918] [General]
![白红色天坛矢量国庆节节日庆祝中文贺卡](https://github.com/user-attachments/assets/6c2a15f3-6dcc-4c71-8218-76702ec5552e)

**Comment by aladdindune** [comment]
支付宝已付款续期1年，使用期限没有延长1年，麻烦解决一下，谢谢.我的邮箱：[aladdindune@hotmail.com]

**Reply by MaxMKMax** [reply]
@aladdindune 您好. 请您也提供我支付单据, 这样我才能帮您提交问题上去. 谢谢您. 祝您假期愉快!

**Comment by bohaixu** [comment]
您好，我在桌面版通过支付宝购买了一年专业版服务，但未收到激活码或类似信息（并且付款渠道中也并没有地方输入邮箱信息）。这是我的邮箱和付款记录，请确认：bohx@mail.ustc.edu.cn
![1728032557992](https://github.com/user-attachments/assets/2bf66cf5-13c0-4fbd-a572-4cdb28f9c0d1)

**Reply by MaxMKMax** [reply]
@bohaixu 您好. 不好意思让您久等了. 上周一进帮您提交上去了, 等待处理. 会尽快给您解决问题.

**Comment by yghpy** [comment]
你好！10月2号付款续费两年，当时到期日未更新，我也没注意。10月5号发现我的专业版到期了，也没有激活码可用，不能使用了，麻烦帮我续上，另外请注意，当时说好赠送3个月的，感谢！我的邮箱 yuenli@hotmail.com
附件是我的支付宝账单，请查看。
![微信图片_20241005095948](https://github.com/user-attachments/assets/f2cbd607-5a43-4722-af55-0519b18342fe)

**Reply by MaxMKMax** [reply]
@yghpy 您好. 不好意思让您久等了. 上周帮您提交上去了, 等待处理. 会尽快给您解决问题.

**Comment by greySkiesMarsh** [comment]
Hello @yghpy, @aladdindune  and @bohaixu 

I've looked into all of your issues and I'm seeing that all of your accounts are currently active and subscribed to Lantern Pro. 

Please follow the login instructions that can be found on our FAQ page if you're having trouble authorizing your devices.

**Comment by aladdindune** [comment]
已经3周都没有解决。请退款！

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
